content/80.glossary.md

```diff
@@ -1,3 +1,7 @@
 ## Glossary
 
-* **Term**: Write the definition as a sentence.
+| Consensus term | Synonyms | Description |
+| -------------- | -------- | ----------- |
+| Network | Graph | A set of *nodes*, representing objects of interest, linked by *edges*, representing specific relationships between nodes. Nodes can have several properties called *attributes* like cell type or position. |
+| Node | Vertice | Element of interest in a network and linked to other nodes. For exemple: people, cells, proteins or genes. |
```

Comment:
```suggestion
| Node | Vertice | Element of interest in a network and linked to other nodes. For example: people, cells, proteins or genes. Nodes can have several properties called *attributes* like cell type or position. |
```
Should `Vertice` be `Vertex` if it's singular?

Reply:
you're right!

Reply:
Ok! Just updated my suggestion so that when you apply it, things will come through - thanks! 😁